### ISSUE TITLE
Error in material_library.pyx: Cannot assign type 'char *' to 'string'

### ISSUE THREAD

**Issue by ljacobson64** [pyne/pyne#1361]
**Describe the Bug**

I'm getting the following error message when building pyne from source:
```
Error compiling Cython file:
------------------------------------------------------------
...

        """
        cdef char * c_filename
        filename_bytes = filename.encode('UTF-8')
        c_filename = filename_bytes
        self._inst.from_json(c_filename)
                                      ^
------------------------------------------------------------

/home/lucas/build/native/pyne/pyne/pyne/material_library.pyx:207:39: Cannot assign type 'char *' to 'string'
```

**To Reproduce**

```
python3 setup.py install --hdf5=/opt/software_native/hdf5-1.10.7 \
                         --moab=/opt/software_native/moab-5.2.0 \
                         --dagmc=/opt/software_native/DAGMC-moab-5.2.0 \
                         --prefix=/opt/software_native/pyne
```

**System information:**
 - OS: Ubuntu 18.04
 - PyNE Version: develop
 - Versions of dependencies installed with: HDF5 1.10.7, MOAB 5.2.0, DAGMC develop

**Comment by nuclearkatie** [comment]
I am also getting this error when trying to build with HDF5-1.12-0. No MOAB, no DAGMC. PyNE on develop, Ubuntu 18.04

**Comment by bam241** [comment]
updating Cython version to `0.29.21` will fix this.

**Comment by gonuke** [comment]
So we consider this closed?  Have we published this minimum version requirement?

**Comment by katyhuff** [comment]
Currently the install instructions say Cython (>= 0.19.1) [here](http://pyne.io/install/source.html#source) and probably elsewhere. 

So, no, I think this remains open until a PR that updates that page.

**Comment by ljacobson64** [comment]
I can confirm that this error stopped occurring after I updated my cython to the latest version.

**Comment by gonuke** [comment]
Closed by #1370
